Set version tag during build

StuBez opened the pull request:
This will be displayed when running sessh with `--version`.

StuBez committed:
Set version tag during build

This will be displayed when running sessh with `--version`.

alanfreer commented on scripts/release.py:
```diff
@@ -16,13 +18,24 @@ def running_on_linux():
+
+    args = parser.parse_args()
+
     add_data_contents = 'config.default.py:.'
```
i think you could perhaps use os.path.sep?

alanfreer commented on scripts/set_version_for_release.py:
```diff
@@ -0,0 +1,22 @@
+    with open(init_path, 'r+') as init_file:
+        lines = init_file.readlines()
+
+        for lineno, line in enumerate(lines):
```
might be a bit clearer as line_no or line_number

StuBez committed:
Use `os.pathsep` for path separator

This saves manually specifying the the path search separator depending
on the operating system that PyInstaller is running on.
